types/servers.pp

```diff
@@ -0,0 +1,4 @@
+type Chrony::Servers = Variant[
```

Comment:
Is there some documentation that makes sense for this type?

Reply:
Yes.  I'll add that. [Puppet-strings style-guide](https://puppet.com/docs/puppet/7.1/puppet_strings_style.html#the-module-readme) wrongly states "However, Strings does not generate information for type aliases or facts"...

Reply:
https://github.com/puppetlabs/puppet-strings/pull/199#issuecomment-754778296